.github/workflows/documentation-commit.yml

```diff
@@ -1,44 +1,35 @@
-# This workflow will install Python dependencies, run tests and lint with a
-# single version of Python. For more information see:
-# https://help.github.com/actions/language-and-framework-guides\
-# /using-python-with-github-actions
+# On each push in 15.0 branch,
+# AND if the coverage file changed,
+# build documentation branch and commit the changes
+# so that the changes are visible on the website
+# https://oca.github.io/OpenUpgrade/
 
 name: Build and commit documentation
 
 on:
   push:
-    branches: ["15.0"]
+    paths: ["docsource/modules140-150.rst"]
 
 jobs:
-  documentation:
-    runs-on: ubuntu-22.04
-    env:
-      DB: "openupgrade"
-      DB_HOST: "localhost"
-      DB_PASSWORD: "odoo"
-      DB_PORT: 5432
-      DB_USERNAME: "odoo"
-      DOWNLOADS: https://github.com/OCA/OpenUpgrade/releases/download/databases
-      ODOO: "./odoo/odoo-bin"
-      PGHOST: "localhost"
-      PGPASSWORD: "odoo"
-      PGUSER: "odoo"
+  documentation-commit:
+    runs-on: ubuntu-latest
```

Comment:
Not sure we should use always the latest version instead of Ubuntu. It can break the CI suddenly.

Reply:
Don't know. Is there a rule, in OCA ? 

in "classic OCA repo", using copier template : 
- for test file, it is ubuntu-latest, [see](https://github.com/OCA/pos/blob/16.0/.github/workflows/test.yml#L14)
- for pre-commit file, it is ubuntu-22.04 [see](https://github.com/OCA/pos/blob/16.0/.github/workflows/pre-commit.yml#L14).

Reply:
It was following this logic: https://github.com/OCA/oca-addons-repo-template/pull/181

Reply:
thanks for the link. 
topic obsolete, as I removed the whole file. But thanks for the information.